CMakeLists.txt

```
@@ -440,8 +440,12 @@ if(CURL_USE_OPENSSL)
   endif()
 
   set(CMAKE_REQUIRED_INCLUDES ${OPENSSL_INCLUDE_DIR})
-  check_symbol_exists(RAND_egd "${CURL_INCLUDES}" HAVE_RAND_EGD)
-  check_symbol_exists(OPENSSL_IS_BORINGSSL "openssl/base.h" HAVE_BORINGSSL)
+  if(NOT (HAVE_RAND_EGD))
```

Shouldn't this be `if(NOT (DEFINED HAVE_RAND_EGD))`?
```suggestion
  if(NOT (DEFINED HAVE_RAND_EGD))
```

---

Thanks for catching this, fixed.